__tests__/app.test.js

```diff
@@ -102,3 +101,30 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("DELETE /api/comments/:comment_id", () => {
+  xtest("204: Responds with message - 'No content", () => {
```

Comment:
We shouldn't be submitting any tests that aren't passing. I assume this is the one you mentioned? Take a look at what you're expecting the body to be, and what the controller is sending back. Do we even need to send anything if we're deleting a post? 🙂